source/common/config/http_subscription_impl.h

```diff
@@ -83,14 +84,15 @@ class HttpSubscriptionImpl : public Http::RestApiFetcher, Config::Subscription<R
       request_.set_version_info(message.version_info());
     } catch (const EnvoyException& e) {
       // TODO(htuch): Track stats and log failures.
+      callbacks_->onConfigUpdateFailed(&e);
     }
   }
 
   void onFetchComplete() override {}
 
   void onFetchFailure(EnvoyException* e) override {
```

Comment:
This is the `RestApiFetcher` interface, so yeah, but maybe outside the scope of this PR.